ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverterImpl.java

```diff
@@ -45,8 +46,16 @@ public EventStreamResponse fromModel(Model model) {
 		final String timestampPath = getResource(model, LDES_TIMESTAMP_PATH);
 		final String versionOfPath = getResource(model, LDES_VERSION_OF);
 		final List<ViewSpecification> views = getViews(model, collection);
-		final String memberType = getResource(model, MEMBER_TYPE);
 		final Model shacl = getShaclFromModel(model);
+		final String memberType = shacl.listStatements(null, shacl.createProperty(targetClass.getURI()), (RDFNode) null)
```

Comment:
If you use `shacl.listObjectsOfProperty(createProperty(targetClass.getURI()))`, you don't need the `null`

Reply:
Amazing! thanks!